addons/account/models/chart_template.py

```diff
@@ -1,997 +1,751 @@
-                'tag_ids': [(6, 0, self.env.ref('account.account_tag_investing').ids)],
-                'company_id': company.id,
-            })
+    def _load(self, template_code, company, install_demo):
+        """Install this chart of accounts for the current company.
 
-        # Set the transfer account on the company
-        company.transfer_account_id = self.env['account.account'].search([
-            ('code', '=like', self.transfer_account_code_prefix + '%'), ('company_id', '=', company.id)], limit=1)
+        :param template_code: code of the chart template to be loaded.
+        :param company: the company we try to load the chart template on.
+            If not provided, it is retrieved from the context.
+        :param install_demo: whether or not we should load demo data right after loading the
+            chart template.
+        """
+        # Ensure that the context is the correct one, even if not called by try_loading
+        if not self.env.is_system():
+            raise AccessError(_("Only administrators can install chart templates"))
+
+        module_names = self.get_chart_template_mapping()[template_code].get('modules', [])
+        module_ids = self.env['ir.module.module'].search([('name', 'in', module_names), ('state', '=', 'uninstalled')])
+        if module_ids:
+            module_ids.button_immediate_install()
+            self.env.reset()  # clear the envs with an old registry
+            self = self.env()['account.chart.template']  # create a new env with the new registry
+
+        self = self.with_context(
+            default_company_id=company.id,
+            allowed_company_ids=[company.id],
+            tracking_disable=True,
+            delay_account_group_sync=True,
+        )
+        company = company.with_env(self.env)
+
+        reload_template = template_code == company.chart_template
+        company.chart_template = template_code
+
+        if not reload_template:
+            for model in ('account.move',) + TEMPLATE_MODELS[::-1]:
+                self.env[model].search([('company_id', '=', company.id)]).with_context({MODULE_UNINSTALL_FLAG: True}).unlink()
+
+        data = self._get_chart_template_data(template_code)
+        template_data = data.pop('template_data')
+
+        if reload_template:
+            self._pre_reload_data(company, template_data, data)
+            install_demo = False
+        data = self._pre_load_data(template_code, company, template_data, data)
+        self._load_data(data)
+        self._post_load_data(template_code, company, template_data)
+
+        # Manual sync because disable above (delay_account_group_sync)
+        AccountGroup = self.env['account.group'].with_context(delay_account_group_sync=False)
+        AccountGroup._adapt_accounts_for_account_groups(self.env['account.account'].search([]))
+        AccountGroup.search([])._adapt_parent_account_group()
+
+        # Install the demo data when the first localization is instanciated on the company
```

Comment:
Does that mean that if I install for example `l10n_be` and then `l10n_nl` I will only get the `l10n_be` demo company?